CMSgov/design-system
# [WNMGDS-1119] svg icon cleanup

## Summary
Phase 2 of icons
This PR contains:
- Accessibility updates for `<SvgIcon>`
- Standardizing sizes for React icon components using ViewBox
- Support for some icon color variations using utility classes
- Lots of documentation around the new icon components (descriptions of each, accessibility features, customization, size & color variations, etc)

## How to test
See the [demo site](http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1119/svg-icon-cleanup/components/icon/)

- Ensure that documentation makes sense. 
- Let me know if any other sections or information could be added
- Look at the icon examples and ensure that sizing looks more consistent

## Outstanding questions
- Does this component needs tests? If so, what kind? I think maybe snapshot tests + accessibility?
